tests/helpers.test.ts

```diff
@@ -1,71 +1,91 @@
 import * as path from 'path';
 
+import { ResponseAnalyzer } from '../src/ResponseAnalyzer';
+
 import {
-    createTestData,
-    parseTextFileContent,
-    calculateSER
-} from '../src/helpers';
+    TranscriptionAnalyzer
+} from '../src/TranscriptionAnalyzer';
 
-import { TranscriptionAnalysisService } from '../src/TranscriptionAnalysisService';
+import { TranscriptionFileService } from '../src/TranscriptionFileService';
 import { TestResult } from '../src/types';
 
-const analyzer = new TranscriptionAnalysisService();
-
 const fakeFileContent = 'sample.wav\tthis is only a sample';
 const fakeAudioFolder = './audio-recording-info'
-const fakeItemInResult = { recording: path.join(fakeAudioFolder, fakeFileContent.split('\t')[0]), transcription: fakeFileContent.split('\t')[1]}
-const testData = parseTextFileContent(fakeFileContent, fakeAudioFolder)
-const TestResultsMock = jest.fn<TestResult, any[]>(()=> ({
-    actualTranscription:'this is only a sample',
+
+const fakeItemInResult = {
+    recording: path.join(fakeAudioFolder, fakeFileContent.split('\t')[0]),
+    transcription: fakeFileContent.split('\t')[1]
+};
+
+const tfs = new TranscriptionFileService();
+const testData = tfs.parseTextFileContent(fakeFileContent, fakeAudioFolder);
+
+const TestResultsMock = jest.fn<TestResult, any[]>(() => ({
+    actualTranscription: 'this is only a sample',
     expectedTranscription: 'this is only a sample',
     wordErrorRate: .023
 }));
-const calculateSERMock = jest.fn<string, any[]>(calculateSER);
+
+const transcriptAnalyzer = new TranscriptionAnalyzer();
```

Comment:
can transcriptAnalyzer just be a static class?

Reply:
I'm not super familiar with static classes. I'll create an issue to remind us to discuss later.